datastream-server-api/src/main/java/com/linkedin/datastream/server/providers/PartitionThroughputProvider.java

```diff
@@ -0,0 +1,31 @@
+/**
+ *  Copyright 2021 LinkedIn Corporation. All rights reserved.
+ *  Licensed under the BSD 2-Clause License. See the LICENSE file in the project root for license information.
+ *  See the NOTICE file in the project root for additional information regarding copyright ownership.
+ */
+package com.linkedin.datastream.server.providers;
+
+import java.util.HashMap;
+
+import com.linkedin.datastream.server.ClusterThroughputInfo;
+
+
+/**
+ * Abstraction that provides topic partition throughput information. Used by load-based assignment strategies to do
+ * partition assignment based on throughput.
+ */
+public interface PartitionThroughputProvider {
```

Comment:
I think the work Partition already implies Topic Partition, because it's not overloaded semantically with different meanings(s) in brooklin. By that same logic we would have to rename `StickyPartitionAssignmentStrategy` to `StickyTopicPartitionAssignmentStrategy`. Not necessary, in my opinion.